picard/acoustid/__init__.py

```diff
@@ -102,24 +105,24 @@ def _lookup_fingerprint(self, next_func, filename, result=None, error=None):
         }
         if not result:
             log.debug(
-                "AcoustID: lookup returned no result for file '%(filename)s'" %
-                mparms
+                    "AcoustID: lookup returned no result for file '%(filename)s'" %
+                    mparms
             )
             self.tagger.window.set_statusbar_message(
-                N_("AcoustID lookup returned no result for file '%(filename)s'"),
-                mparms,
-                echo=None
+                    N_("AcoustID lookup returned no result for file '%(filename)s'"),
+                    mparms,
+                    echo=None
             )
             file.clear_pending()
             return
         log.debug(
-            "AcoustID: looking up the fingerprint for file '%(filename)s'" %
-            mparms
+                "AcoustID: looking up the fingerprint for file '%(filename)s'" %
```

Comment:
Actually i think it is a bad mix.
Either we align on first char after the `(` or we wrap all and indent with 4 spaces.
I prefer the second approach, because it allows longer lines (less indentation spaces), and allow to keep args aligned (one per line), which is much more readable.
It limits impact of changes (modification of an argument name, adding/remove args, etc...) which makes diffs more easier to read (diffs are line-oriented mostly).

My preferred style, especially with many args:
```python
somefunc(
    "the string %s" %
    arg
)
```

Or, in certain cases (only 2 lines):
```python
somefunc("the string %s" %
         arg)
```